web/packages/extension/manifest.json

```diff
@@ -14,7 +14,7 @@
     "content_scripts": [
         {
             "matches": ["<all_urls>"],
-            "js": ["js/lv0.js"],
+            "js": ["lv0.js"],
```

Comment:
This seems to break the extension for me; the build process still places the js file in `js/lv0.js`.